core/src/main/java/io/opencensus/stats/MeasurementMap.java

```diff
@@ -18,34 +18,34 @@
 import java.util.NoSuchElementException;
 
 /**
- * A map from {@link MeasurementDescriptor}'s to measured values.
+ * A map from {@link Measure}'s to measured values.
  */
 public final class MeasurementMap implements Iterable<MeasurementValue> {
```

Comment:
This will be done in another PR on refactoring `MeasurementValue`, and that PR will be built on this one. I didn't put all changes together so that I can keep each PR relatively small :p